### ISSUE TITLE
build fail

### ISSUE THREAD

**Issue by marcorosa** [Storj/python-libstorj#4]
`./build.sh` output:

```
building '_python_libstorj' extension
x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/include/python2.7 -c lib/ext/python_libstorj_wrap.cxx -o build/temp.linux-x86_64-2.7/lib/ext/python_libstorj_wrap.o
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
In file included from lib/ext/python_libstorj_wrap.cxx:3165:0:
lib/ext/python_libstorj.h:1:19: fatal error: storj.h: No such file or directory
compilation terminated.
error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

**Comment by bryanchriswhite** [comment]
@marcorosa do you have libstorj installed? https://github.com/storj/libstorj

I now realize it's not in the readme.. i'll add it.

**Comment by bryanchriswhite** [comment]
This should be addressed by [improve devops (#2)](https://github.com/bryanchriswhite/python-libstorj/pull/2)

**Comment by marcorosa** [comment]
Fixed by #PR2 on your branch.
I'm gonna close this issue
